tap_github/authenticator.py

```diff
@@ -61,7 +66,7 @@ def is_valid_token(self) -> bool:
             return True
         except requests.exceptions.HTTPError:
             msg = (
-                f"A token was dismissed. "
+                f"A token could not be validated. "
```

Comment:
Downgrading this statement since this method now gets called multiple times.  Added another `"A token was dismissed."` log message down below at the point of dismissal (L247), so it'll still appear in the log when tokens are dismissed during initial authentication.